kratos/includes/mortar_classes.h

```diff
@@ -1319,20 +1319,24 @@ class DualLagrangeMultiplierOperators
      */
     boost::numeric::ublas::bounded_matrix<double, TNumNodes, TNumNodes> CalculateAe()
```

Comment:
missing a description of what it does

Reply:
There is a description in the Doxygen standard